src/Orient/Orient.Client/Configuration.cs

```diff
@@ -25,7 +30,19 @@ static Configuration()
         /// <value>
         /// The timeout in msec.
         /// </value>
-        public static int Timeout { get; set; }
+        public static int Timeout
+        {
+            get
+            {
+                return timeout;
+            }
+
+            set

```

Comment:
Setting this from the type constructor would drop and establish all connections.